### PR TITLE
Fixes #365: Added steps in documentation

### LINKED ISSUE #365: Add a page 'how to add Susper as search provider in Chrome browser' in tutorials
## Problem
We will be adding up ' how to add susper as search provider in chrome' in tutorials section following to DuckDuckGo - https://duckduckgo.com/install?t=hc because Chrome has some different feature to add search provider. Catch up more discussion here - #314 

## Solution
One can start working on this once, **search-settings** page component has been created in #270

### PR DESCRIPTION
**Pull request by harshit98**
Fixes issue #365 

Changes: I have added gif file demonstrating how to add Susper as default search engine in Google Chrome.

Demo Link: <a href="https://github.com/harshit98/susper.com/blob/yay/README.md">here</a>

Screenshots for the change: 
(Not appropriate here)

Kindly review @Marauderer97 @nikhilrayaprolu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,3 +148,7 @@ npm install
 ## Further help
 
 To get more help on the `angular-cli` use `ng --help` or go check out the [Angular-CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
+
```
Comment (nikhilrayaprolu) [inline README.md]: there is a lot not useful info or wrong info on susper in README, Can you remove all those in this pull.

**Comment by harshit98** [comment]
@nikhilrayaprolu done!

**Comment by nikhilrayaprolu** [comment]
Yoi have understood me wrong. There are other lines in susper regarding old susper-backbone (beautiful css lines and all), the starting intro could be cut down much, like the lines about the ajax technology and all is not required There is a line mentioning bower but we aren't using it anywhere. There are some more lines that need to be removed.

**Comment by harshit98** [comment]
@nikhilrayaprolu Kindly review. Let me know if any more changes to be done.

**Pull request merged**